containers/java-postgres/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+{
+	"name": "Java & PostgreSQL",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "app",
+	"workspaceFolder": "/workspace",
+	
+	// Set *default* container specific settings.json values on container create.
+	"settings": {
+		"sqltools.connections": [
+			{
+			  "name": "Container database",
+			  "driver": "PostgreSQL",
+			  "previewLimit": 50,
+			  "server": "localhost",
+			  "port": 5432,
+			  // NOTE: database/username/password should match values in docker.compose.yml
+			  "database": "postgres",
+			  "username": "postgres",
+			  "password": "postgres"
+			}
+		  ],
+		"java.home": "/docker-java-home"
+	},
+	
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"vscjava.vscode-java-pack",
+		"mtxr.sqltools",
+		"mtxr.sqltools-driver-pg"
+	],
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [5432],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
```

Comment:
Do we need those comments? 

The 1st one is explained in `README.md`, the 2nd one is not. Take out or explain in `README.md`?

Reply:
`postCreateCommand` is a common Docker feature and the comments here are again taken from the existing templates for other `devcontainer.json` files. I'm not sure this actually needs more explanation, since the comment outlines exactly what the command would do.

Reply:
Best practice is to not have unused code, Please remove.

Reply:
I opted to add info about `postCreateCommand` to the README.md, so the details are more 'upfront' for how to use it.